Google.PowerShell/Compute/GceAttachedDiskConfigCmdlets.cs

```diff
@@ -134,14 +135,23 @@ protected override void ProcessRecord()
                 Source = Source?.SelfLink
             };
 
+            if(DiskType != null && DiskType.Contains("local-ssd"))
```

Comment:
We should use the `GetUriPart` function inherited from `GcloudCmdlet` to extract out the type of the disk. A call to `GetUriPart("disktype", DiskType.ToLower())` should return `local-ssd`.

Reply:
Ok, will fix and update.